.NET/Microsoft.Recognizers.Text.Number/Parsers/BaseNumberParser.cs

```diff
@@ -26,6 +26,16 @@ public BaseNumberParser(INumberParserConfiguration config)
                                 GetKeyRegex(this.Config.CardinalNumberMap.Keys) + "|" +

```

Comment:
I suggest we extract this to a private method - BuildTextNumberRegex(). 
From `var singleIntFrac` to 'TextNumberRegex = new Regex(textNumberPattern, RegexOptions.Singleline | RegexOptions.Compiled);'. The new method can return the right regex for assignment here.

Reply:
@tellarin, we updated the files

Reply:
Not quite what I meant for the comments, but ok. :)
```
/* 
 * It could look
 * like this
 */
```